<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CleanupFilesWindow</class>
 <widget class="QWidget" name="CleanupFilesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cleanup Files</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbl_status">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="toolTip">
      <string>Right-click to export</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Remove all</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>CleanupFilesWindow</receiver>
   <slot>_remove_all()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_remove_all()</slot>
 </slots>
</ui>
